### ISSUE TITLE
Modify the assembly version from edit assembly attributes did not work

### ISSUE THREAD

**Issue by chucklu** [0xd4d/dnSpy#1235]
- What version did you use?
dnSpy v6.0.5 (.NET Framework)

- Write down all steps needed to reproduce this issue. Also attach any needed files.
[Hearthbuddy-20180802002.zip](https://github.com/0xd4d/dnSpy/files/3467018/Hearthbuddy-20180802002.zip)
I have modified the assembly version and file version to 2018.8.2.2, however it still show the old version in FileExplorer.
![image](https://user-images.githubusercontent.com/3004057/62456457-06082b00-b7ab-11e9-8e8b-67090dc58262.png)
![image](https://user-images.githubusercontent.com/3004057/62456549-3cde4100-b7ab-11e9-974a-320e2b26b2cc.png)
The detail info also show the old version 0.3.1466.417 .
![image](https://user-images.githubusercontent.com/3004057/62456566-49629980-b7ab-11e9-9538-c34138680213.png)

**Comment by chucklu** [comment]
Also tried to get the version through Powershell
```Powershell
> (Get-Item Hearthbuddy-20180802002.exe).VersionInfo.FileVersion
0.3.1446.417
```

**Comment by chucklu** [comment]
The same problem with  copyright 
[assembly: AssemblyCopyright("Copyright ©  2019")]

**Comment by 0xd4d** [comment]
Yes, there's a bunch of attributes that get written to win32 version info.
